gapii/cc/spy_base.cpp

```diff
@@ -52,7 +52,7 @@ void SpyBase::init(CallObserver* observer) {
   mIsSuspended = false;
 }
 
-void SpyBase::lock(CallObserver* observer) { mSpinLock.Lock(); }
+void SpyBase::lock() { mSpinLock.Lock(); }
```

Comment:
Commit message should mention that you're doing this :)

Reply:
Done!